pkg/k8s/table.go

```diff
@@ -36,6 +39,9 @@ func (tw TableWriter) Write(report Report) error {
 	case summaryReport:
 		writer := NewSummaryWriter(tw.Output, tw.Severities)
 		return writer.Write(report)
+	default:
+		return xerrors.Errorf("report %q not supported. please use \"summary\" or \"all\"", tw.Report)
```

Comment:
Not a must, but can be used to avoid escaping:

```suggestion
		return xerrors.Errorf(`report %q not supported. please use "summary" or "all"`, tw.Report)
```